test/benchmark/config/httproute.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: "{HTTPROUTE_NAME}"
+  namespace: benchmark-test
+spec:
+  parentRefs:
+    - name: "{REF_GATEWAY_NAME}"
+  hostnames:
+    - "www.benchmark.com"
```

Comment:
can we also template out the hostname for this test so each HTTPRoute gets a unique hostname
else these routes wont reach `Programmed` and will bloat `Status`

Reply:
make sense, do it as a follow-up